### PR TITLE
Add article premium attribute

### PR DESCRIPTION
**Pull request by andygout**
cc @andygnewman

### CODE REVIEW

File: test/server/models/article-v3-spec.js [hunk 1]
```diff
@@ -171,6 +171,11 @@ describe('Article v3 model', function () {
 			subject.webUrl.should.equal(fixtureV1.item.location.uri);
 		});
 
+		it.only('uses article\'s assigned webUrl value to ascertain and assign premium status', function () {
```
Comment (matthew-andrews) [inline test/server/models/article-v3-spec.js]: no only……

**Comment by matthew-andrews** [comment]
i'm not sure this is the right place for this data…  we have an API that tells you the content level…

https://content-classification-api.ft.com/content/69dc8904-0008-11e6-99cb-83242733f755

**Comment by andygout** [comment]
The API URL goes to a blank page... Have tried some other article `uuid`s but no joy.

**Comment by matthew-andrews** [comment]
look at the headers ;)

**Comment by matthew-andrews** [comment]
screen shot 2016-09-19 at 12 10 42

**Comment by matthew-andrews** [comment]
+1

**Pull request merged**